net-im/qtox/qtox-1.3.0.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit eutils qmake-utils
+
+DESCRIPTION="Most feature-rich GUI for net-libs/tox using Qt5"
+HOMEPAGE="https://github.com/tux3/qtox"
+SRC_URI="https://github.com/tux3/qTox/archive/v${PV}.tar.gz"
```

Comment:
Consider renaming the tarball to avoid a potential distfile collision with another package: ``v${PV}.tar.gz -> ${P}.tar.gz``.